codespell_lib/_codespell.py

```diff
@@ -35,20 +35,30 @@
 """
 VERSION = '2.0.dev0'
 
+supported_languages_en = ('en', 'en_GB', 'en_US', 'en_CA', 'en_AU')
+supported_languages = supported_languages_en
+
 # Users might want to link this file into /usr/local/bin, so we resolve the
 # symbolic link path to the real path if necessary.
 _data_root = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
 _builtin_dictionaries = (
     # name, desc, name, err in aspell, correction in aspell
     # The aspell tests here aren't the ideal state, but the None's are
     # realistic for obscure words
-    ('clear', 'for unambiguous errors', '', False, None),
-    ('rare', 'for rare but valid words', '_rare', None, None),
-    ('informal', 'for informal words', '_informal', True, True),
-    ('usage', 'for recommended terms', '_usage', None, None),
-    ('code', 'for words common to code and/or mathematics', '_code', None, None),  # noqa: E501
-    ('names', 'for valid proper names that might be typos', '_names', None, None),  # noqa: E501
-    ('en-GB_to_en-US', 'for corrections from en-GB to en-US', '_en-GB_to_en-US', True, True),  # noqa: E501
+    ('clear', 'for unambiguous errors', '',
+        False, None, supported_languages_en, None),
+    ('rare', 'for rare but valid words', '_rare',
+        None, None, None, None),
+    ('informal', 'for informal words', '_informal',
+        True, True, supported_languages_en, supported_languages_en),
+    ('usage', 'for recommended terms', '_usage',
+        None, None, None, None),
+    ('code', 'for words common to code and/or mathematics', '_code',
+        None, None, None, None,),  # noqa: E501
+    ('names', 'for valid proper names that might be typos', '_names',
+        None, None, None, None,),  # noqa: E501
+    ('en-GB_to_en-US', 'for corrections from en-GB to en-US', '_en-GB_to_en-US',  # noqa: E501
+        True, True, ('en_GB',), ('en_US',)),  # noqa: E501
```

Comment:
I think you can drop most of these noqa's now you've line wrapped so they're shorter.

Reply:
I still have to get used to this python/flake stuff :-)